frontend/.env.local.example

```diff
@@ -1,2 +1,2 @@
-REACT_APP_POST_URL_PORT=":3000"
+REACT_APP_POST_URL_PORT="3000"
```

Comment:
pls add a new line, even github suggests you that - to soothe my OCD

Reply:
[here](https://github.com/salsita/foosball-rating/commit/eca8143fe7504de0175d2608f3b8d0f0f03bd9e7#diff-8c572d46d23eaf9d608eec00f24c16a3)